css/properties.json

```diff
@@ -4152,6 +4153,30 @@
     ],
     "status": "standard"
   },
+  "gap": {
+    "syntax": "<'row-gap'> <'column-gap'>?",
+    "media": "visual",
+    "inherited": false,
+    "animationType": [
+      "row-gap",
+      "column-gap"
+    ],
+    "percentages": "no",
+    "groups": [
+      "CSS Box Alignment"
+    ],
+    "initial": [
```

Comment:
Should be 
```
"initial": [
      "row-gap",
      "column-gap"
]
```

Reply:
I was thinking about it, but documentation on w3c says something different. Please check...

Reply:
Looks like a mistake in the spec. A shorthand should have the same initial values as its corresponding properties.
We need to fill out an issue on the problem in the spec's tracker.

Reply:
Fixed on my side. Not sure how to fill out an issue on the problem in the spec's tracker.

Reply:
You can find a link to issue tracker in the beginning of spec: https://github.com/w3c/csswg-drafts/labels/css-align-3
It's hosted on GitHub, just create an issue with prefix `[css-align] ` and describe a problem: `gap` is a shorthand for `row-gap` and `column-gap`, but its initial value differ with initial values of those properties.

Reply:
Done https://github.com/w3c/csswg-drafts/issues/2504

Reply:
Should we merge this PR in?

Reply:
Here is an answer on your issue:
> You are right, it should say "see individual properties". This has already been corrected in the latest draft: https://drafts.csswg.org/css-align/#gap-shorthand

I was right 😎So we can merge it without any hesitation

Reply:
Yeay! First we need to merge #188 - can u have a look at it?

Reply:
#188 merged so this can go in too!